wpeverest/user-registration
# UR-801 Uniform boolean setting value saving and retrieval

### All Submissions:






### Changes proposed in this Pull Request:

Previously, boolean setting values( checkbox, toggle ) were saved in many different formats like Yes/No or 1/0 or True/False. 
Due to that we are getting confused when retrieving these values as to which format the value is stored.

So, this PR stores and retrieves all these values in uniform format. (1|0)
Jira Link: [UR-801](https://themegrill.atlassian.net/browse/UR-801)

### How to test the changes in this Pull Request:

1. Switch to the PR branches in all User Registration and addons.
2. Without any changes, verify if all the features with checkbox settings are working properly and checkbox values are set properly according to the previous values.
3. Try saving the settings page.
4. Verify if the boolean values are stored as ( 0 | 1 ) in the database.
5. Again, verify if all the features linked with these boolean settings ( checkbox and toggle ) are working properly.

### Types of changes:




### Other information:




### Changelog entry

> UR-801 Uniform boolean setting value saving and retrieval


[UR-801]: https://themegrill.atlassian.net/browse/UR-801?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
